Examples/Algorithms/HepMC/src/HepMCProcessExtractor.cpp

```diff
@@ -134,7 +134,7 @@ std::vector<ActsExamples::SimParticle> selectOutgoingParticles(
       simParticle.setPosition4(pos4.x(), pos4.y(), pos4.z(), pos4.t());
       Acts::Vector3 mom3(mom4[0], mom4[1], mom4[2]);
       simParticle.setDirection(mom3.normalized());
-      simParticle.setAbsMomentum(mom3.norm());
+      simParticle.setAbsoluteMomentum(mom3.norm());
```

Comment:
Do both of these methods exist, or why did it work before and after the change?

Reply:
Only `.setAbsoluteMomentum` is available now. It was broken before but we did not see it since it was not part of the CI build. This is fixed with this PR.

Reply:
Ok